x/bank/client/cli/sendtx.go

```diff
@@ -24,7 +21,7 @@ const (
 // SendTxCmd will create a send tx and sign it with the given key.
 func SendTxCmd(cdc *codec.Codec) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "send",
+		Use:   "send [to_address] [amount]",
```

Comment:
Since we are here we might make `--from` turn into an argument

Reply:
I don't think we should do that. Would require many more changes. Also would be nice to set a default `from` in the config